### PR TITLE
Add script changes for VertexAI

### PR DESCRIPTION
**Pull request by morganchen12**
Let me know if the release note should be more specific or if the version should be something else.

### CODE REVIEW

File: FirebaseVertexAI/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# 0.2.0
+
+- [added] Initial VertexAI release.
```
Comment (paulb777) [inline FirebaseVertexAI/CHANGELOG.md:3]: Can we add a link to the announcement and detailed docs?

Comment (morganchen12) [reply FirebaseVertexAI/CHANGELOG.md]: Announcement TBD, but I added a link to the docs.

File: FirebaseVertexAI/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# 0.2.0
```
Comment (andrewheard) [inline FirebaseVertexAI/CHANGELOG.md:1]: I think _this_ should be `10.26.0`. @paulb777 do you know for sure?

**Review by paulb777 — APPROVED** [review]

File: FirebaseVertexAI/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+
+- [added] Initial VertexAI release. See the
```
Comment (andrewheard) [inline FirebaseVertexAI/CHANGELOG.md:3]: Maybe something like this?
```
- [added] Public Preview release of the Vertex AI for Firebase SDK - build
  AI-powered mobile apps and features with the Gemini API using this new SDK.
  See the
  [docs](https://firebase.google.com/docs/vertex-ai/get-started?platform=ios)
  for instructions on how to get started.
```

**Review by andrewheard — APPROVED** [review]

File: FirebaseVertexAI/CHANGELOG.md (hunk 2 above)
Comment (paulb777) [reply FirebaseVertexAI/CHANGELOG.md:1]: Yes. Good catch.

File: FirebaseVertexAI/CHANGELOG.md (hunk 3 above)
Comment (morganchen12) [reply FirebaseVertexAI/CHANGELOG.md]: Done.

File: FirebaseVertexAI/CHANGELOG.md (hunk 2 above)
Comment (morganchen12) [reply FirebaseVertexAI/CHANGELOG.md]: Done.

**Pull request merged**